bedrock/security/templates/security/bug-bounty/faq-webapp.html

```diff
@@ -97,9 +97,35 @@ <h2>Eligible bugs</h2>
         </dd>
       </dl>
 
-      <h2>Bug reporting</h2>
+      <h2 id="bug-reporting">Bug reporting</h2>
 
       <dl class="faq">
+        <dt id="what-next">Once I have found a vulnerability, what next?</dt>
+        <dd>
+          <p>The sooner we can reproduce and fix the bug, the sooner we can
+            fix it and send you your bounty. Please submit all bugs through
+            the <a href="https://bugzilla.mozilla.org/form.web.bounty">Bugzilla web bounty form</a>.
+            <strong>Do not send vulnerabilities via email and please avoid using video.</strong></p>
```

Comment:
Joke: " ... using video or we will find you" ;)

Reply:
"We know who you are."